docker-compose.yaml

```diff
@@ -18,6 +18,8 @@ services:
     container_name: grafana
     image: grafana/grafana-oss:9.4.3
     volumes:
+      # Note: The 'ro' option specifies that the volume is mounted in read-only mode.
+      # However, certain files such as grafana.ini may require write access. Be mindful of security implications when removing ':ro'.
```

Comment:
Could you please put the doc link here as reference?

Reply:
fyi, you could also put `Fixes: #7` in your commit message, so that it would close #7 automatically once the PR get merged.

ref: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue